src/components/MainLayout/MainLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { Navigation } from '../Navigation/Navigation';
+
+export const MainLayout: React.FC = () => {
+  return (
+    <>
+      <nav
+        data-cy="nav"
+        className="navbar is-fixed-top has-shadow"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="container">
+          <div className="navbar-brand">
```

Comment:
You already have these tags inside `Navigation`

<img width="465" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/86789161/1b8d9088-86f8-4342-abf3-f8d99b6669e4">

Reply:
> You already have these tags inside `Navigation`
> 
> <img alt="image" width="465" src="https://user-images.githubusercontent.com/86789161/272484472-1b8d9088-86f8-4342-abf3-f8d99b6669e4.png">

Still not fixed, you have the same nav inside nav that doesn't make any sense

Reply:
Now there are two identical navigations in the application
<img width="888" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/54860641/fde43718-7ad5-4676-823d-ce66e56f855d">